pkg/sqlite/load_test.go

```diff
@@ -131,7 +131,7 @@ func TestAddPackageChannels(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.description, func(t *testing.T) {
 			db := fmt.Sprintf("%d.db", rand.Int())
-			store, err := NewSQLLiteLoader(db)
+			store, err := NewSQLLiteLoader(db, "./db_migrations")
```

Comment:
We should probably be testing the compiled migrations, right?

Reply:
We do further down in the file:

```
migrator, err := NewSQLLiteMigrator(store.db, "")
```

But actually testing the values of the default migrations would involve either keeping the latest version in a configuration file or actually parsing the version. At that point, we would be maintaining test code that provides no functionality except to execute tests: IMO that is unnecessary overhead and technical debt.

That's why I implemented all the value checking using a set of test migrations that can be statically defined and their values referenced directly in the tests, and the actual check of the real data is just "does this throw an error when parsed".

Do you feel like there is significant value to be gained from ensuring something I'm missing?

Reply:
Ah, just realized this is for `load_test.go`, not the new test file I added. Yeah in this case you are right this should be updated to just use the correct default path.